mavros/src/plugins/local_position.cpp

```diff
@@ -75,10 +80,11 @@ class LocalPositionPlugin : public MavRosPlugin {
 
 	ros::NodeHandle pos_nh;
 	ros::Publisher local_position;
-	tf::TransformBroadcaster tf_broadcaster;
+	boost::shared_ptr<tf::TransformBroadcaster> tf_broadcaster;
```

Comment:
@tonybaltovski You can revert this please. And make listener normal too, without pointer.